config/settings.yml

```diff
@@ -40,6 +41,8 @@ SCIENCEWIRE:
   PORT: 443
   RECOMMENDATION_PATH: /PublicationCatalog/MatchedPublicationItemIdsForAuthor?format=xml
   TMPDIR: /tmp
+  ARTICLE_BASE_URL: https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/
+  DOI_BASE_URL: https://dx.doi.org/
```

Comment:
Should split this Setting out into a separate group for `DOI`, like `DOI.BASE_URL`?  The DOI system stands alone, it is not SW service.